app/src/main/java/ar/com/wolox/android/training/utils/Utils.java

```diff
@@ -0,0 +1,18 @@
+package ar.com.wolox.android.training.utils;
+
+import android.app.Activity;
+import android.content.Intent;
+
+import ar.com.wolox.android.training.ui.login.LoginActivity;
+
+public class Utils {
```

Comment:
Maybe we can call this UserHelper , UserUtils or SessionUtils . Just to give the message to other developers